VideoWeb/VideoWeb/ClientApp/src/app/services/api/video-web.service.ts

```diff
@@ -1,140 +1,145 @@
+        return this.apiClient.addSelfTestFailureEventToConference(conferenceId, addSelfTestFailureEventRequest).toPromise();
+    }
+
+    getPexipConfig(): Promise<SelfTestPexipResponse> {
+        return this.apiClient.getPexipConfig().toPromise();
+    }
+
+    getObfuscatedName(displayName: string): string {
+        return displayName.replace(/(?!\b)\w/g, '*');
+    }
+
+    getDistinctJudgeNames(): Promise<JudgeNameListResponse> {
+        return this.apiClient.getDistinctJudgeNames().toPromise();
+    }
+
+    /**
+     * Get the chat history where sender/reciver is from/to given username in a conference
+     * @param conferenceId conference Id
+     * @param participantUsername participant's username to filter chat history
+     */
+    getConferenceChatHistory(conferenceId: string, participantUsername: string): Promise<ChatResponse[]> {
+        return this.apiClient.getConferenceInstantMessageHistoryForParticipant(conferenceId, participantUsername).toPromise();
+    }
+
+    /**
+     * Get a total of unread messages betwen admin and all partcipants in a conference
+     * @param conferenceId conference id
+     */
+    getUnreadMessageCountForConference(conferenceId: string): Promise<UnreadInstantMessageConferenceCountResponse> {
+        return this.apiClient.getNumberOfUnreadAdminMessagesForConference(conferenceId).toPromise();
+    }
+
+    /**
+     * Get the total of unread message between an admin and given username
+     * @param conferenceId conference id
+     * @param participantUsername participant's username to filter chat history
+     */
+    getUnreadMessagesForParticipant(conferenceId: string, participantUsername: string): Promise<UnreadAdminMessageResponse> {
+        return this.apiClient.getNumberOfUnreadAdminMessagesForConferenceByParticipant(conferenceId, participantUsername).toPromise();
+    }
+
+    setActiveIndividualConference(conference: ConferenceForIndividualResponse) {
+        const conf = new ConferenceLite(conference.id, conference.case_number);
+        this.activeConferencesCache.clear();
+        this.activeConferencesCache.set(conf);
+    }
+
+    getActiveIndividualConference(): ConferenceLite {
+        return this.activeConferencesCache.get();
+    }
+
+    getParticipantsWithContactDetailsByConferenceId(conferenceId: string): Promise<ParticipantContactDetailsResponseVho[]> {
+        return this.apiClient.getParticipantsWithContactDetailsByConferenceId(conferenceId).toPromise();
+    }
+
+    updateParticipantDetails(
+        conferenceId: string,
+        participantId: string,
+        updateParticipantRequest: UpdateParticipantRequest
+    ): Promise<void> {
+        return this.apiClient.updateParticipantDisplayName(conferenceId, participantId, updateParticipantRequest).toPromise();
+    }
+
+    getCourtRoomsAccounts(venueAllocation: string[]): Promise<CourtRoomsAccountResponse[]> {

```

Comment:
Since this is only for the VHO command centre, keep this with `VhoQueryService`

Reply:
moved to vhoQueryService, done